WTWSMS/localisation/WtWSMS_warfare.csv

```diff
@@ -26,7 +26,8 @@ RETTYPE_CUL_ROMAN;Legio Comitatensis;Legio Comitatensis;;;;;;;;;;;;x
 RETTYPE_CUL_ILLY;Balkan Falxmen;Guerriers balkaniques � falx;;;;;;;;;;;;x
 RETTYPE_CUL_RBT;Brito Bellatorum;Brito Bellatorum;;;;;;;;;;;;x
 RETTYPE_CUL_GMR;Germanorum Peditum;Germanorum Peditum;;;Germanorum Peditum;;;;;;;;;x
-RETTYPE_CUL_PAR;Teutonicus Foederatus;Teutonicus Foederatus;;;Federados Teut�nicos;;;;;;;;;x
+#RETTYPE_CUL_PAR;Teutonicus Foederatus;Teutonicus Foederatus;;;Federados Teut�nicos;;;;;;;;;x
+RETTYPE_CUL_PAR;Cataphract;;;;;;;;;;;x
```

Comment:
As discussed.

Reply:
Right, I forgot about the retinues. I  will split them into two subunits tomorrow.

Reply:
OK, done. Split into Cataphract and Aswaran